nix/pkgs.nix

```diff
@@ -25,9 +25,20 @@ let
 
   # Override some packages and utilities
   pkgsOverlay = import ./overlay.nix;
+
+  # Fix for lack of Android SDK for M1 Macs.
+  systemOverride = let
+    inherit (builtins) currentSystem getEnv;
+    systemOverride = getEnv "NIXPKGS_SYSTEM_OVERRIDE";
```

Comment:
Oh, that's not good, I didn't notice this. We have two variables that are the same. My bad.

Lets call it `systemEnvVar` or something.